src/containers/Group/GroupList/GroupListCard/GroupListCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Should we abstract this to 'ListCard' component? We can use this for other listings too.